tests/util/test_object.py

```diff
@@ -1,7 +1,47 @@
 ''' Tests for datamegh.util.object util. '''
 
 import pytest
-from datamegh.util.object import linearize, delinearize, merge
+from datamegh.util.object import linearize, delinearize, merge, dict_to_list
+
+
+def test_dict_to_list_1():
+    '''
+    Test a dictionary converted to list.
+    '''
+    value = {
+        'foo': 'bar',
+        'just': 'test',
+        'hello': 'world',
+    }
+
+    expected = [
+        {'name': 'foo', 'value': 'bar'},
+        {'name': 'just', 'value': 'test'},
+        {'name': 'hello', 'value': 'world'}
+    ]
+
+    assert dict_to_list(value) == expected
+
+def test_dict_to_list_2():
+    '''
+    Test an empty dictionary converted to empty list.
+    '''
+    value = {}
+
+    expected = []
+
+    assert dict_to_list(value) == expected
+
+def test_dict_to_list_3():
+    '''
+    Test an invalid argument such as int to dict_to_list
+    '''
+    value = 1
+
+    with pytest.raises(AttributeError) as ex:
+        dict_to_list(value)
+
+    assert ex.value.args[0] == "'int' object has no attribute 'items'"
```

Comment:
Good that you have this test that catches error. 
I think we'll need to catch this type error inside and throw our own specific error from the function if the type is not a dictionary. Here's what it should say.

```py
    assert ex.value.args[0] == "Argument must be a dictionary, invalid argument received '1'."
```